packages/auth/src/grant/routes.ts

```diff
@@ -416,12 +416,12 @@ async function deleteGrant(
     ctx.throw(401, { error: 'invalid_request' })
   }
   const grant = await deps.grantService.getByContinue(continueId, continueToken)
-  if (!grant) {
+  if (!grant || grant.state === GrantState.Revoked) {
     ctx.throw(404, { error: 'unknown_request' })
   }
 
-  const deletion = await deps.grantService.deleteGrant(continueId)
-  if (!deletion) {
+  const revoked = await deps.grantService.revokeGrant(continueId)
```

Comment:
I suppose we don't need `revokeGrant` (by `continueId`) since we have the grant id here as well. Both `revokeGrant` and `revokeGrantById` existed already as `deleteGrant` and `deleteGrantById`. I dont see the harm in changing it to one `revokeGrant` function that takes the grant id.

Reply:
changed to just do `revokeGrant(grantId)`